Apps/Common/SIMSolver.h

```diff
@@ -143,8 +193,15 @@ template<class T1> class SIMSolver : public SIMadmin
     if (saveRes && !S1.saveStep(tp,nBlock))
       return false;
 
-    if (saveRes && exporter)
-      exporter->dumpTimeLevel(&tp,newMesh);
+    if (saveRes && exporter) {
+      DataExporter::SerializeData serializeData;
+      if (exporter->getRestartStride() > 0 &&
+          tp.step % exporter->getRestartStride() == 0) {
+        S1.serialize(serializeData);
+        this->serialize(serializeData);
+      }
```

Comment:
`S1.serialize(serializeData);` kan med fordel flyttes inne i `SIMSolver::serialize` i istedet, så holder det med `this->serialize` her. Spesiellt hvis du er enig i at det er en god ide med en `TimeStep::serialize`også (se forrige kommentar),